Kong/docs.konghq.com
# Third party plugins cleanup: Yes Interactive plugins

### Description

Removing plugins maintained by Yes Interative as part of the third-party plugin cleanup initiative.
* KongMap
* Kong Log Google
* Jwt2Header


### Testing instructions

Netlify link: 
https://deploy-preview-5452--kongdocs.netlify.app/hub/yesinteractive/kongmap
https://deploy-preview-5452--kongdocs.netlify.app/hub/yesinteractive/kong-jwt2header
https://deploy-preview-5452--kongdocs.netlify.app/hub/yesinteractive/kong-log-google
 should redirect to /hub/, and the plugins should be gone from the landing page.


### Checklist
